### PR TITLE
Correcting note box for measure_fock

### PR DESCRIPTION
**Pull request by antalszava**
Details on diff: `..note` -> `.. note` and removing `:code:` for rendering.

### CODE REVIEW

File: strawberryfields/backends/base.py [hunk 1]
```diff
@@ -33,11 +33,11 @@ def __init__(self, num_subsystems):
         self._init = num_subsystems
         #: list[int]: _map[k] is the internal index used by the backend for
         # computational mode k, or None if the mode has been deleted
-        self._map = [k for k in range(num_subsystems)]
+        self._map = list(range(num_subsystems))
```
Comment (antalszava) [inline strawberryfields/backends/base.py:36]: Making Codefactor happy with linting

File: strawberryfields/backends/base.py [hunk 2]
```diff
@@ -386,9 +387,10 @@ def measure_homodyne(self, phi, mode, shots=1, select=None, **kwargs):
     def measure_fock(self, modes, shots=1, select=None, **kwargs):
         """Measure the given modes in the Fock basis.
 
-        ..note::
-          When :code:``shots == 1``, updates the current system state to the conditional state of that
-          measurement result. When :code:``shots > 1``, the system state is not updated.
+        .. note::
```
Comment (antalszava) [inline strawberryfields/backends/base.py:390]: Correcting the typo

File: strawberryfields/backends/base.py (hunk 1 above)
Comment (josh146) [reply strawberryfields/backends/base.py:36]: agree with this change 😆

File: strawberryfields/backends/base.py (hunk 2 above)
Comment (josh146) [reply strawberryfields/backends/base.py:391]: Good 👁️ !

**Pull request merged**